README.md

```diff
@@ -39,6 +39,47 @@ Or to some custom path:
 ```
 make PREFIX=/some/custom/path install
 ```
+## Ubuntu (and probably Debian)
+
+Install nvim, prerequisites, Rust + Cargo.
+
+* Use rustup.rs -- https://rustup.rs
+* This example has to be done as root since the rustup installer installs cargo in one's home dir
+
+Example (Tested on Ubuntu 23.04)
+
+``` shell
+#become root
+sudo su -
```

Comment:
That's really bad advice to promote on documentation.  Besides `sudo -s` is for that purpose.

Regardless, you should never recommend a user to become full administrator of their system.